Backpack-SwiftUI/Switch/Classes/BPKSwitch.swift

```diff
@@ -36,9 +36,13 @@ public struct BPKSwitch<Content: View>: View {
     /// - Parameter isOn: A binding to a property that indicates whether the switch is
     ///    on or off.
     /// - Parameter text: A string that describes the purpose of the switch.
-    public init(isOn: Binding<Bool>, text: String) where Content == BPKText {
+    /// - Parameter truncate: A Boolean value that determines whether the text should be truncated
+    ///    or displayed on multiple lines.
+    public init(isOn: Binding<Bool>, text: String, truncate: Bool = true) where Content == BPKText {
+        let bpkText = truncate ? BPKText(text) : BPKText(text).lineLimit(nil)
```

Comment:
I think it could be slightly easier like 
```suggestion
        BPKText(text).lineLimit(truncate ? 1 : nil)
```
But it's not critical, both variants should work fine 👍